NeuralNetwork/network.h

```diff
@@ -24,5 +24,5 @@ struct network_group
 
 	void set_t_network_size(T_Network &network);
 
-	void network_group::modify_vector(Network & net_1, float mutation_chance);
-};
\ No newline at end of file
+	void modify_vector(Network & net_1, float mutation_chance);
```

Comment:
Because it's already in the struct, you don't need (and actually according to the standard, can't have) the extra `network_group::`. Yet MSVC allows it.